evap/results/views.py

```diff
@@ -230,15 +230,23 @@ def user_can_see_text_answer(user, represented_users, text_answer, view):
     if text_answer.is_private:
         return contributor == user
 
+    # NOTE: when changing this behavior, make sure all changes are also reflected in results.tools.textanswers_visible_to
+    # and in results.tests.test_tools.TestTextAnswerVisibilityInfo
     if text_answer.is_published:
+        # text answers about responsible contributors can only be seen by the users themselves and by their delegates
+        # they can not be seen by other responsible contributors
         if text_answer.contribution.responsible:
-            return contributor == user or user in contributor.delegates.all()
+            return contributor in represented_users
```

Comment:
please check again, i'm pretty sure this is not equivalent. previously it was `user in contributor.represented_users`, now it is `contributor in user.represented_users`. if the code in the PR is currently incorrect, please make a test demonstrating the problem.

Reply:
you're right, it's correct.